src/Files/WeeklyFilePattern.php

```diff
@@ -31,10 +31,10 @@ public static function fromInfo(array $info, \DateTimeInterface $now = NULL) {
     ];
     $interval = new \DateInterval('P1W');
 
-    $end = $now->modify('+1 day')->modify('last monday')->setTime(0, 0, 0, 0);
+    $end = $now->modify('+1 day');
     $start = $now->sub($info['retention_period'])->modify('-1 day')->modify('next monday')->setTime(0, 0, 0, 0);
-    if ($info['include_current']) {
-      $end = $end->add($interval);
+    if (!$info['include_current']) {
+      $end = $end->sub($interval);
```

Comment:
Doesn’t this exclude a bit too much now? I'd expect "current" to be the current week starting on Monday. E.g. today is Wednesday and I don’t include current, I'd expect the export to still include Sunday last, not stop on Wednesday? Something more like `$end = $end->modify('last monday')->setTime(0, 0, 0, 0);`?

Reply:
In theory it does. The `DatePeriod` object that’s built is only used for the start dates though. So this means in your example the `DatePeriod` would still issue the Monday last week as a file and therefore would export all the data up to and including Sunday.

Reply:
So only the Mondays between start and end date matter but will still export the entire week following them?! Then I guess this works ;)